__tests__/app.test.js

```diff
@@ -0,0 +1,47 @@
+const request = require("supertest");
+const app = require("../app.js");
+const connection = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+const { string } = require("pg-format");
```

Comment:
This might be an auto import but can be removed - we don't need pg-format in the test file.